src/States/Elytra.cs

```diff
@@ -466,9 +473,11 @@ private static void Mod_Player_UpdateSprite(On.Celeste.Player.orig_UpdateSprite
                 frame -= (int)(t * (FRAME_COUNT - STABLE_FRAME - 1));
             else
                 frame -= (int)(t * STABLE_FRAME);
-        }
-
-        self.Sprite.SetAnimationFrame(Calc.Clamp(frame, 0, FRAME_COUNT - 1));
+        }
+        if ((self.Sprite.CurrentAnimationTotalFrames - 1) >= frame)

```

Comment:
```suggestion
        if (frame < self.Sprite.CurrentAnimationTotalFrames)

```

Tiny nit, even though I still prefeare limiting it via the `Clamp` without the `if`.